### PR TITLE
bpo-37548: Document range of atan, acos and asin

### PR DESCRIPTION
**Pull request by potomak**
Document the range of the inverse trigonometric functions `math.atan`,
`math.asin` and `math.acos`.

https://bugs.python.org/issue37548

### CODE REVIEW

File: Doc/library/math.rst [hunk 1]
```diff
@@ -366,17 +366,20 @@ Trigonometric functions
 
 .. function:: acos(x)
 
-   Return the arc cosine of *x*, in radians.
+   Return the arc cosine of *x*, in radians. The result is between ``-1.0`` and
```
Comment (mdickinson) [inline Doc/library/math.rst]: I think this should say: "between 0 and pi"

Comment (potomak) [reply Doc/library/math.rst]: Yes, sorry for the silly mistake. :(

File: Modules/mathmodule.c [hunk 2]
```diff
@@ -1072,19 +1072,22 @@ math_2(PyObject *const *args, Py_ssize_t nargs,
 
 FUNC1(acos, acos, 0,
       "acos($module, x, /)\n--\n\n"
-      "Return the arc cosine (measured in radians) of x.")
+      "Return the arc cosine (measured in radians) of x. "
+      "The result is between 0 and pi")
```
Comment (mdickinson) [inline Modules/mathmodule.c]: One nitpick: please could you add full stops to the end of the sentences (here and for `asin` and `atan`, too)?

**Review by mdickinson — CHANGES_REQUESTED** [review]
One nitpick; apart from that, LGTM. Thank you!

File: Modules/mathmodule.c [hunk 3]
```diff
@@ -1072,19 +1072,22 @@ math_2(PyObject *const *args, Py_ssize_t nargs,
 
 FUNC1(acos, acos, 0,
       "acos($module, x, /)\n--\n\n"
-      "Return the arc cosine (measured in radians) of x.")
+      "Return the arc cosine (measured in radians) of x. "
```
Comment (ZackerySpytz) [inline Modules/mathmodule.c]: I don't think there should be a trailing space after the period (the same goes for the other docstrings being modified). Also, I think the period should be followed by `\n\n` in order to conform to [PEP 257](https://www.python.org/dev/peps/pep-0257/) (like the docstring for `atan2()`).
```suggestion
      "Return the arc cosine (measured in radians) of x.\n\n"
```

**Comment by potomak** [comment]
@mdickinson not a b locker, but I didn't notice 29be072 and accidentally overwritten it (commit --amend + push -f). I'm not sure why, but I can't find a way to recover it from either upstream nor origin.

**Comment by mdickinson** [comment]
@potomak No problem; I've re-applied that commit. In general there's not much benefit to force-pushing for PRs against this repository: everything gets squashed at merge time anyway. (More information on this [in the dev-guide](https://devguide.python.org/pullrequest/?highlight=force#submitting).)

**Review by mdickinson — APPROVED** [review]
LGTM!

**Comment by potomak** [comment]
@mdickinson 

> No problem; I've re-applied that commit.

Thanks :)

> In general there's not much benefit to force-pushing for PRs against this repository: everything gets squashed at merge time anyway.

Good to know, thanks for the info. I will review the dev-guide more carefully.

**Pull request merged**